tests/torch/ptq/test_graphs.py

```diff
@@ -36,7 +36,7 @@
 @pytest.fixture(scope='function', params=ALGOS)
 def graph_dir(request):
     quantization_type = request.param
-    graph_dir = os.path.join('quantized', quantization_type)
+    graph_dir = Path('quantized') / 'ptq' / quantization_type
```

Comment:
Unfortunately, the current PTQ architecture is not compatible with reusing the PT `QuantizationBuilder` code. The PTQ algos for PT had to copy-and-paste code and do not produce the same result as the `create_compressed_model` approach. @daniil-lyakhov was offering to refactor `QuantizationBuilder` to make the quantizer creation more in-line with the `ModelTransformer` approach, and when that is done, we can make the test stronger by requiring the match to the quantized graphs of the QAT approach.